### PR TITLE
hud: move tiltfile to the top

### PR DESCRIPTION
**Pull request by nicks**
Hello @yuindustries, @landism,

Please review the following commits I made in branch nicks/ch2203/tiltfile:

683ce40d634a624cc05776193ff47a490d19ce2b (2019-04-09 18:01:22 -0400)
hud: move tiltfile to the top

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2019-04-09 18-01-58

**Comment by nicks** [comment]
i could go either way on this, but it fixes the immediate issue

File: web/src/Sidebar.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ class SidebarItem {
   name: string
   status: string
   hasWarnings: boolean
+  hasEndpoints: boolean
```
Comment (landism) [inline web/src/Sidebar.tsx]: is this supposed to be in a different branch?

Comment (nicks) [reply web/src/Sidebar.tsx]: nope! now that the tiltfile is at the top, it's not very helpful to open a preview of the tiltfile by default

**Pull request merged**